arangod/RocksDBEngine/RocksDBIterators.h

```diff
@@ -82,6 +82,9 @@ class RocksDBAnyIndexIterator final : public IndexIterator {
   void resetImpl() override;
 
  private:
+  template <typename Func>
+  bool doNext(size_t limit, Func func);
```

Comment:
Should be pass by const reference to avoid copying here?
```suggestion
  bool doNext(size_t limit, Func const& func);
```

Reply:
Sure, although the lamdas of the two callers are small enough that this shouldn't make any difference.